### DISCUSSION TITLE
Консультация по дизайну

### DISCUSSION THREAD

**Discussion by Totohka** [Totohka/GreenTeam#42] [Ideas]
Консультация для Дениса Усольцева от Эдуарда Новикова

**Comment by DISLIKERA** [comment]
Ссылки на добавление в проект в Figma: https://www.figma.com/file/H8zvLLfMMKBDaMdArZJici/%D0%B3%D0%BB%D0%B0%D0%B2%D0%BD%D0%B0%D1%8F%2C-%D0%BC%D0%B5%D0%BD%D1%8E-%D0%BE%D0%BF%D1%86%D0%B8%D0%B9%2C-%D0%BA%D0%BE%D1%80%D0%B7%D0%B8%D0%BD%D0%B0%2C-%D0%BA%D0%B0%D1%80%D1%82%D0%BE%D1%87%D0%BA%D0%B0-%D1%82%D0%BE%D0%B2%D0%B0%D1%80%D0%B0?type=design&node-id=2%3A21&mode=design&t=lYYZrtuMUz3cknbk-1

https://www.figma.com/file/SqxnXzC1PMOXZU1P1phw1w/%D1%80%D0%B5%D0%B3%D0%B8%D1%81%D1%82%D1%80%D0%B0%D1%86%D0%B8%D1%8F%2C-%D0%B2%D1%85%D0%BE%D0%B4%2C-%D0%BB%D0%B8%D1%87-%D0%BA%D0%B0%D0%B1%D0%B8%D0%BD%D0%B5%D1%82%2C-%D0%B8%D0%BD%D1%84%D0%B0-%D0%BE-%D0%BC%D0%B0%D0%B3%D0%B0%D0%B7%D0%B8%D0%BD%D0%B5?type=design&node-id=1%3A3&mode=design&t=HgetbMUZNlbAwoiZ-1

**Comment by Nedofff** [comment]
В целом это скорее рекомендации и пища для ума, чем что-то неоспоримое.

# Общее
- Не вижу причин использовать несколько проектов в фигме
    - Так как это один проект то всем кто будет пользоваться дизайном будет удобнее с одной страницы все брать)
- Имхо было бы круто создать палитру цветов [тык](https://www.youtube.com/watch?v=gJY2V-3sxoA)
    - Если надо будет что-то менять, то ты легко это сделаешь изменив одно значение
    - Верстальщики и фронтендеры легко смогут вынести цвета в переменные css и также легко изменять и использовать - будет достигнута единообразность приложения и дизайна так сказать
- Про формы округление углов это в целом круто и добавляет привлекательности и позитивного настроя дизайну, но ты слегка перебарщиваешь
    - Такая большая округленность начинает привлекать внимание и из-за этого пользователь теряется между кучей округленных углов, нам бы не хотелось, чтоб пользователь смотрел на углы
    - Для строгости или четкого горизонтального и вертикального баланса можно даже оставить острые углы - попробуй поиграть с этим, но опять же не перебарщивай с округлением (точно до 32px)
    - Я так понимаю, что в карточках товара более темный цвет обозначает фото товара, так что тут советую быть осторожным и почитать [эту статейку](https://habr.com/ru/articles/789314/) про скругление внутренних углов
- Про цвета
    - Салатовый очень блеклый - он не добавляет интереса и не влияет на настрой от сайта - по сути это цвет для разбавления яркости
        - Советую увеличить насыщенность (около 80%), чтоб он добавлял больше настроя. Зеленый, кстати, говорит о мирности и здоровье и я не могу точно сказать, что это то, что нужно магазину, если он не продает лекарства, БАДы или какие-нибудь травы
        - Так как конкретного товара нет, то выбрать основной цвет сложно, однако просто ниже перечислю связь эмоций и цветов, которые могут быть интересны шаблонному магазину
        - Серый = сбалансированный и спокойствие/покой
        - Оранжевый = дружелюбный, жизнерадостный, уверенный
        - Желтый = оптимистичный, прозрачный, теплый
- Для фото и различных элементов как цена стоит использовать реальные - возможно, что придется переделывать, когда появятся реальные - данные
- По хорошему нужно сделать еще версию под мобильные устройства, так как там много что по-другому
- Интересный пример цветов и форм - [тык](https://www.behance.net/gallery/165581903/SIRE-Website-UXUI-Concept?tracking_source=search_projects%7Cweb+shop+white&l=12)
### Шапка
- Иконки ну типа дохлые
- ![image](https://github.com/Totohka/GreenTeam/assets/78249138/cf66ddc0-4792-4a96-bf15-cdccf2c0ef2a)
- Использовать градиент стоит с идеей, с правильными цветами - тут надо почитать статейки
- Шапка почти всегда однотонна
    - Если шапка однотонная, то часто хватает изменения цвета и можно не использовать тени
# Регистрация, вход, лич кабинет, инфа о магазине
## Регистрация
- Лучше либо фон формы сделать темнее, чтобы инпуты были сильнее выделены
- Либо вообще убрать фон и добавить теней или границу в один пиксель
    - Инпуты можно в таком случае оставить такого же цвета как были или сделать тонкую границу
    - Граница это черный с прозрачностью на 25%
    - С прозрачным фоном черный текст будет читабельнее
- Сейчас любят делать с текстом в инпутах вот так: ![image](https://github.com/Totohka/GreenTeam/assets/78249138/eeaf4977-5854-4659-b0e5-57ced3a8f897)
- Либо можно сделать текст более прозрачным без “:”, чтоб дальше это было просто плейсхолдерами, то есть исчезло при вводе
- Стоит продумать, что будет происходить при наведении и нажатии на кнопку
- Тут тот же момент, что и в общем - пусть округление будет в рамках от 5 до 20, максимум 40
    - Но кнопка может быть такой округленной с ней все ок
- Надо подумать, что сделать с цветом кнопки и текста на ней - они плохо контрастируют, что не очень хорошая практика
    - Можно сделать текст белым, а кнопку темнее/зеленее
    - Или использовать тени без фона
    - Или темно серый цвет кнопки и белый текст, что меняется наоборот при наведении
## Вход
- Во входе все тоже самое, что и при регистрации
- В личном кабинете по сути также как и в регистрации, но
    - Можно разделить представление информации и чеки - допустим, информация слева, а чеки справа, иначе очень маленькое окошко для чеков
    - Еще советовал бы не использовать слайдер для чеков, а сделать ленту
        - Слайдер не очень работает, когда в нем много элементов, а чеков может быть как 1, так 100500
    - Также можно добавить фильтры, например, по дате для чеков
## Инфа о нас
- Не знаю как по макету, но подобная информация (которая справа) размещается в футере на каждой странице 
    - Хотя адрес и контакты можно накидать в разных местах главной страницы для того, чтоб можно было быстро найти магазин и тп
    - Но это зависит от формата магазина, а у вас нет по сути формата, так что не знаю стоит ли раскидывать по всяким страницам - без идеи сложно понять конечный результат
- Документы тоже можно убрать как ссылку в футер или в меню

# Главная, меню опций, корзина, карточка товара
## Главная
- В целом это задача дизайнера, так что тут стоит как-то выделить карточку товара, если пользователь наводиться на нее
    - Ozon подсвечивает шрифты, а WB добавляет теней
## Меню опций
- Блок слишком широк
- Текст плохо контрастирует с фоном
- Тени от фона категории товара налазят друг на друга
- [Вот примеры](https://www.dejurka.ru/web-design/side-menu-web-design/) различных боковых меню - вдохновись 
##Корзина, карточка товара
- Стоит сделать отступы от цифры и кнопок +/-
- Цвет “Адрес доставки” не очень сочетается с салатовым - [здесь](https://colorscheme.ru/) есть основные цветовые схемы и лучше использовать их
    - И в целом лучше использовать все цвета в каких-то равных пропорциях, а не единожды на сайте, чтоб создать айдентику - набор выдержанных в едином стиле и концепции визуальных и прочих элементов, которые формируют образ и восприятие бренда, а использования цвета единожды излишне
- У “Адрес доставки” есть тени, а у “Заказать” нет
    - Лучше в целом придерживаться одной идеи и я не уверен, что тени нужны раз ты выделяешь с помощью фона
- В карточке товара как будто слишком много пустого пространства, но хз
